private-templates-service/adaptivecards-templating-service/package.json

```diff
@@ -16,12 +16,15 @@
   },
   "license": "ISC",
   "dependencies": {
+    "adaptivecards-templating": "^0.1.1-alpha.1",
     "axios": "^0.19.2",
     "bunyan": "^1.8.12",
     "express": "^4.17.1",
     "express-validator": "^6.4.0",
+    "install": "^0.13.0",
     "jws": "^4.0.0",
     "mongoose": "^5.8.10",
+    "npm": "^6.14.4",
```

Comment:
I am not sure you intended to include these ?

Reply:
I did not, thanks. Removed them.